core/pfe-sass/mixins/_custom-properties.scss

```diff
@@ -190,6 +190,12 @@
 ///   // ...as well as all the values for :host([on="saturated"])  and  :host([on="light"])
 @mixin pfe-contexts($contexts: $CONTEXTS) {
   // Helper attributes to aid in theming
+
+  :host([context="dark"]) { --context: dark; }
+  :host([context="light"]) { --context: light; }
+  :host([context="accent"]) { --context: saturated; }
+  :host([context="saturated"]) { --context: saturated; }
+
```

Comment:
How was this working before without this? 🤔

Reply:
🙈

Reply:
real answer: [javascript](https://github.com/patternfly/patternfly-elements/blob/0eb2f08d5e78f61104b0503a360e96aaec5a3613/elements/pfelement/src/pfelement.js#L286)